src/commands/image/imageSource/EnvFileListStep.ts

```diff
@@ -73,6 +73,14 @@ export class EnvFileListStep<T extends EnvFileListContext> extends AzureWizardPr
         return context.imageSource !== ImageSource.QuickstartImage && context.environmentVariables === undefined;
     }
 
+    public confirmationViewProperty(context: T): ConfirmationViewProperty {
+        return {
+            name: localize('environmentVariables', 'Environment Variables'),
+            value: context.envPath ?? localize('useExisting', 'Use existing configuration'),
+            contextPropertyName: 'envPath'
```

Comment:
This can be a future thing, but I was thinking that we should make this type a key of the context. That way the only accepted values would be properties that actually exist on the context and prevent any typos, would provide auto-complete, etc.

Similar to how we do that with `nonNullProp`